quickannotator/dl/training.py

```diff
@@ -122,7 +123,7 @@ def train_pred_loop(config):
 
     running_loss = []
     
-    writer = SummaryWriter(log_dir=f"/tmp/{classid}/{datetime.datetime.now().strftime('%b%d_%H-%M-%S')}")
+    writer = SummaryWriter(log_dir=f"/tmp/{annotation_class_id}/{datetime.datetime.now().strftime('%b%d_%H-%M-%S')}")
```

Comment:
these should go somewhere in the project directory and not /tmp

Reply:
Okay. I'm going to set up the output directory structure in the export functionality PR (not yet opened).

I've noted this request.